src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1531,7 +1531,7 @@ internal override ImmutableArray<Symbol> GetEarlyAttributeDecodingMembers(string
 
                 if (!membersAndInitializers.HaveIndexers)
                 {
-                    membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(s => s.Name);
+                    membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(static s => s.Name);
```

Comment:
>ToDictionary [](http://example.com/codeflow?start=74&length=12)

It looks like the change that used pooled accumulator in this helper got reverted. Was this intentional?

Reply:
Yes.  I thought that would go in in an independent PR.  I can pull back into this though.

Reply:
Added back.